tests/unit/test_logger.py

```diff
@@ -0,0 +1,19 @@
+"""Test logging in config.py."""
+
+import pytest
+
+from wristpy.core import config
+
+
+def test_get_logger(caplog: pytest.LogCaptureFixture) -> None:
+    """Test the wristpy logger with level set to 20 (info)."""
+    logger = config.get_logger()
+
+    logger.debug("Debug message here.")
+    logger.info("Info message here.")
+    logger.warning("Warning message here.")
+
+    assert logger.getEffectiveLevel() == 20
+    assert "Debug message here" not in caplog.text
```

Comment:
If we change to DEBUG level, I believe this will have to change to

Reply:
That's fine - if we do the test will fail and it can be modified at that point in time.